packages/react/src/components/ItemLayout/ItemLayout.tsx

```diff
@@ -62,12 +62,10 @@ export interface ItemLayoutProps extends UIComponentProps, ContentComponentProps
   contentMediaCSS?: ICSSInJSStyle
   /** Styled applied to the end media element of the rendered component. */
   endMediaCSS?: ICSSInJSStyle
-  truncateContent?: boolean
-  truncateHeader?: boolean
 }
 
 /**
- * The Item Layout handles layout styles for menu items, list items and other similar item templates.
+ * (DEPRECATED) The Item Layout handles layout styles for menu items, list items and other similar item templates.
```

Comment:
Can we suggest alternative approach here?  Or point to Layout guide?

Reply:
it should be enough to reference alternative components to use - may suggest not to include any links to the descriptions, as those are subject to potential changes

Reply:
done in #1154